onyo/main.py

```diff
@@ -179,7 +190,7 @@ def setup_parser() -> argparse.ArgumentParser:
         'mv',
         description=commands.mv.__doc__,
         formatter_class=SubcommandHelpFormatter,
-        help=commands.mv.__doc__
+        help='Move assets or directories into to a destination directory; or rename a directory.'
```

Comment:
"... into to ..." I think one of those might be enough :)

Reply:
Thanks for catching this! Fixed and pushed.